docs/README.md

```diff
@@ -412,7 +412,7 @@ To remove a price table, select the option you want to remove from the **Assigne
 
 ##### Users
 
-This section presents a list of users associated with the organization, showing their **Email**, their **Role** in the organization and the **Cost Center** they are associated with.
+This section presents a list of users associated with the organization, showing their **emails**, their **roles** within the organization, and the **cost center** to which they are associated. It is worth noting that the same email can be associated with up to 949 different organizations or cost centers.
```

Comment:
Why 949? How did you get to this number?

Reply:
Rodolfo Bussola created [this card](https://vtex-dev.atlassian.net/browse/EDU-12886) requesting the addition of this information to the documentation. Is this information incorrect?

Reply:
Hmm I'm not sure honestly, I will talk to Rodolfo. But anyway, better to have this upper limit explicit here than no limit at all.